### DISCUSSION TITLE
Implement `skimage.segmentation.clear_border` in ImageMagick

### DISCUSSION THREAD

**Discussion by 00sapo** [ImageMagick/ImageMagick#6753] [Digital Image Processing]
What is the best way to implement [this](https://scikit-image.org/docs/stable/api/skimage.segmentation.html#skimage.segmentation.clear_border) function in ImageMagick?

**Comment by snibgo** [comment]
You could do that in a program or shell script. `-connected-components` lists the bounding box of each component. Where the top, bottom, left or right of a bounding box is on the image edge, part of the component is on the image edge. `-connected-components` also labels the pixels of each component, so you can change those pixels to whatever you desire.
